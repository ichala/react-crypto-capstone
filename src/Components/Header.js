import React from 'react';
import {
  Link,
} from 'react-router-dom';

function Header() {
  return (
    <div className="col-12 d-flex justify-content-between p-3" style={{ backgroundColor: 'black' }}>
      <Link to="/">
        {' '}
        <span className="text-white">Home</span>
        {' '}
      </Link>
      <span className="text-white">MetaCrypt</span>
      <span className="text-white  d-flex  ">
        <i className="bi bi-mic-fill" />
        <i className="bi bi-gear-fill" />
      </span>
    </div>
  );
}

export default Header;
