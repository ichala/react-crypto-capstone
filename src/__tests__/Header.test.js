import { Provider } from 'react-redux';
import { HashRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Header from '../Components/Header';
import store from '../Redux/configureStore';

it('Home Components Renders correctly', () => {
  const tree = renderer.create(
    <Provider store={store}><Router><Header /></Router></Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
