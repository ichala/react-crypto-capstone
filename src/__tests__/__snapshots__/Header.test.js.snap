// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Components Renders correctly 1`] = `
<div
  className="col-12 d-flex justify-content-between p-3"
  style={
    Object {
      "backgroundColor": "black",
    }
  }
>
  <a
    href="#/"
    onClick={[Function]}
  >
     
    <span
      className="text-white"
    >
      Home
    </span>
     
  </a>
  <span
    className="text-white"
  >
    MetaCrypt
  </span>
  <span
    className="text-white  d-flex  "
  >
    <i
      className="bi bi-mic-fill"
    />
    <i
      className="bi bi-gear-fill"
    />
  </span>
</div>
`;
